hack/deploy_cincinnati.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
```

Comment:
probably want the usual:

```sh
set -o nounset
set -o errexit
set -o pipefail
```

boilerplate, to avoid blithely ignoring failures.